### PR TITLE
restructure oadp docs

### PR DESCRIPTION
**Pull request by shubham-pampattiwar**
- Added docs folder
- Updated readme
- Added NooBaa docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,9 @@ OADP is OpenShift Application Data Protection operator. This operator sets up an
 - OpenShift CLI
 - Access to OpenShift cluster
 
-## Getting Started
+***
+## Getting Started with basic install
```
Comment (dymurray) [inline README.md:14]: nit:

>Getting started with basic install

(lowercase s)

File: README.md [hunk 2]
```diff
@@ -108,205 +110,49 @@ replicaset.apps/velero-76546b65c8          1         1         1         2m17s
...
-   The above specification will install Velero with 3 plugins (azure, gcp and custom-plugin-example).
+***
+## Customize Installation
+***
 
-### Enable CSI plugin for Velero
+### Plugin customization
 
-By default the CSI plugin is not enabled, in order to enable the [CSI plugin](https://github.com/vmware-tanzu/velero-plugin-for-csi/) for velero, you need to specify a flag `enable_csi_plugin` and set it to `true` in the `konveyor.openshift.io_v1alpha1_velero_cr.yaml` file during the installation.
+During the Velero installation using OADP Operator we need at least one plugin. Please refer [Velero plugin customization](docs/plugins.md) for more details.
```
Comment (dymurray) [inline README.md:119]: Nit: The Velero installation requires at least one cloud provider plugin installed. Please refer ...

File: README.md (hunk 1 above)
Comment (shubham-pampattiwar) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (shubham-pampattiwar) [reply README.md]: Done, updated the readme

File: README.md [hunk 3]
```diff
@@ -108,205 +110,49 @@ replicaset.apps/velero-76546b65c8          1         1         1         2m17s
...
-    config:
-      region: us-west-2
-      profile: "default"
-```
-<b>Note:</b> 
-- Be sure to use the same `secret` name you used while creating the cloud credentials secret in step 3 of Operator   installation section.
-- Another thing to consider are the CR file specs, they should be tailored in accordance to your own cloud provider accounts, for instance `bucket` spec value should be according to your own bucket name and so on.
-- Do not configure more than one `backupStorageLocations` per cloud provider, the velero installation will fail.  
-- Parameter reference for [backupStorageLocations](https://velero.io/docs/master/api-types/backupstoragelocation/) and [volumeSnapshotLocations](https://velero.io/docs/master/api-types/volumesnapshotlocation/)
+Velero supports backup storage locations and volume snapshot locations from a number of cloud providers (AWS, Azure and GCP). Please refer the section [configure Backup Storage Locations and Volume Snapshot Locations](docs/bsl_and_vsl.md).
```
Comment (dymurray) [inline README.md:127]: nit: Please refer to the section

**Pull request merged**